syntax = "proto3";

package proto;

option go_package = "./go";

service TextGenService {
    // Метод для генерации текста
    rpc GenerateText (GenerateRequest) returns (GenerateResponse);
}

message GenerateRequest {
    string question = 1;
}

message GenerateResponse {
    string answer = 1;
}